apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/container/edit.js

```diff
@@ -275,7 +274,10 @@ function Edit( props ) {
 				onError( props, result.message );
 			}
 		);
-		props.selectBlock();
+
+		// Execution delayed with setTimeout to ensure it runs after any block auto-selection performed by inner blocks
+		// (such as the Recurring Payments block)
+		setTimeout( () => props.selectBlock(), 1000 );
```

Comment:
This looks fragile, but I don't have any better suggestions, without understanding the codebase more. If we could yield this function to its dependent actions, that would be ideal.

Reply:
Good point! It could be a followup if anyone likes, I'm merging to get the original issue solved.